transcriptic/auth.py

```diff
@@ -34,13 +37,21 @@ def __init__(self, email, secret):
             )
 
     def __call__(self, request):
+        if urlparse(request.url).netloc != urlparse(self.api_root).netloc:
+            return request
```

Comment:
Would be good to add a single-line comment for documentation

Reply:
Will do 👍 . Thanks for reviewing.

Reply:
Extracted to method and moved to base class. Should be clear what is going on by the method & variable names, but let me know if you think otherwise.